CHANGES.rst

```diff
@@ -15,6 +15,8 @@
 
 - Moved ma_table_name and ma_table_number from observation to exposure groups. [#74]
 
+- Update pin min version pin for astropy to 5.0.4. [#75]
```

Comment:
too many 'pin's?

Reply:
Too many of these PRs my brain was on autopilot.